source/connectors/kafka-sink-data-formats.txt

```diff
@@ -9,10 +9,331 @@ Kafka Sink Connector Data Formats
+       },
+       {
+         "name": "age",
+         "type": "int"
+       },
+       {
+         "name": "active",
+         "type": "boolean"
+       },
+       {
+         "name": "address",
+         "type": {
+           "type": "record",
+           "name": "AddressRecord",
+           "fields": [
+             {
+               "name": "city",
+               "type": "string"
+             },
+             {
+               "name": "country",
+               "type": "string"
+             }
+           ]
+         }
+       },
+       {
+         "name": "food",
+         "type": {
+           "type": "array",
+           "items": "string"
+         }
+       },
+       {
+         "name": "data",
+         "type": {
+           "type": "array",
+           "items": {
+             "type": "record",
+             "name": "Whatever",
+             "fields": [
+               {
+                 "name": "k",
+                 "type": "string"
+               },
+               {
+                 "name": "v",
+                 "type": "int"
+               }
+             ]
+           }
+         }
+       },
+       {
+         "name": "lut",
+         "type": {
+           "type": "map",
+           "values": "double"
+         }
+       },
+       {
+         "name": "raw",
+         "type": "bytes"
+       }
+     ]
+   }
+
+
+.. _avro-logical-types-example:
+
+AVRO Logical Type Example
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following AVRO schema demonstrates use of each supported logical types
+(``Decimal``, ``Date``, ``Time``, and ``Timestamp``):
+
+.. code-block:: json
+
+   {
+     "type": "record",
+     "name": "MyLogicalTypesRecord",
+     "namespace": "com.mongodb.kafka.data.kafka.avro",
+       "fields": [
+         {
+           "name": "myDecimalField",
+           "type": {
+             "type": "bytes",
+             "logicalType": "decimal",
+             "connect.parameters": {
+               "scale": "2"
+             }
+           }
+         },
+         {
+           "name": "myDateField",
+           "type": {
+             "type": "int",
+             "logicalType": "date"
+           }
+         },
+         {
+           "name": "myTimeMillisField",
+           "type": {
+             "type": "int",
+             "logicalType": "time-millis"
+           }
+         },
+         {
+           "name": "myTimeMicrosField",
+           "type": {
+             "type": "long",
+             "logicalType": "time-micros"
+           }
+         },
+         {
+           "name": "myTimestampMillisField",
+           "type": {
+             "type": "long",
+             "logicalType": "timestamp-millis"
+           }
+         },
+         {
+           "name": "myTimestampMicrosField",
+           "type": {
+             "type": "long",
+             "logicalType": "timestamp-micros"
+           }
+         }
+       ]
+     }
+
+.. note::
+
+   If you use compile your schema using AVRO, it maps your logical types
```

Comment:
Looks like a small grammar mistake got introduced. Also, maybe "AVRO compiles your schema types to the following non-standard Java classes" ? Not a hard nit except for the grammar at the beginning of the sentence.